src/libs/ReportUtils.ts

```diff
@@ -4616,11 +4616,23 @@ function buildOptimisticMoneyRequestEntities(
     return [createdActionForChat, createdActionForIOUReport, iouAction, transactionThread, createdActionForTransactionThread];
 }
 
+// Check if the report is empty report
```

Comment:
We should use comments to be descriptive and explain _why_ or _how_ the code is doing so and so, instead of using comments to repeat what the code is already saying. 

So here, we can say something like:

>// Check if the report is empty, meaning it has no visible messages (i.e. only a "created" report action).